src/main/java/com/celements/appScript/AppScriptService.java

```diff
@@ -25,17 +39,33 @@ public class AppScriptService implements IAppScriptService {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AppScriptService.class);
 
-  @Requirement
-  IEmptyCheckRole emptyCheck;
+  @Inject
+  private IEmptyCheckRole emptyCheck;
```

Comment:
better use constructor injection. also change to spring `@Component`.

Reply:
Out of scope of this issue. Thus added https://synjira.atlassian.net/browse/CELDEV-1197